src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -1,56 +1,65 @@
-  <div id="addEditPersonContainer">
-    <confirm-dialog ref="reassignCPDialog" attach="#addEditPersonContainer" />
+  <div id="add-edit-org-person">
+
+    <confirm-dialog
+      ref="reassignCPDialog"
+      attach="add-edit-org-person"
+    />
 
     <v-expand-transition>
       <ul class="list add-person">
-        <li class="add-person-container">
+        <li class="add-person-container px-8 py-12">
           <div class="meta-container">
 
-            <!-- *** TODO: split into 2 columns -->
-            <label class="add-person-header" v-if="isOrg && entityFilter(CorpTypeCd.BENEFIT_COMPANY)">
+            <label class="add-org-header" v-if="isOrg && entityFilter(CorpTypeCd.BENEFIT_COMPANY)">
               <span v-if="activeIndex === -1">Add Corporation or Firm</span>
               <span v-else>Edit Corporation or Firm</span>
             </label>
 
-            <label class="add-person-header" v-if="isOrg && entityFilter(CorpTypeCd.COOP)">
+            <label class="add-org-header" v-if="isOrg && entityFilter(CorpTypeCd.COOP)">
               <span v-if="activeIndex === -1">Add Organization</span>
               <span v-else>Edit Organization</span>
             </label>
 
-            <!-- *** TODO: implement message box -->
-            <!-- <div class="message-box" v-if="isCompletingParty">
-              <p>
-                <strong>Important:</strong> The Completing Party information below is based on your
-                BC Registries account information. Your name cannot be changed here. Name changes must
-                be made through your account settings.
-              </p>
-              <p>
-                If you make changes to your address below, please update your address in the account
-                settings after you have completed thisfiling to ensure your information is up to date.
-              </p>
-            </div> -->
+            <label class="add-org-header" v-if="isPerson">
+              <span v-if="activeIndex === -1">Add Person</span>
+              <span v-else>Edit Person</span>
+            </label>
 
             <div class="meta-container__inner">
+              <v-card outlined class="message-box" v-if="isCompletingParty && !isRoleStaff">
+                <p>
+                  <strong>Important:</strong> The Completing Party information below is based on your
+                  BC Registries account information. Your name cannot be changed here. Name changes must
+                  be made through your account settings.
+                </p>
+                <p>
+                  If you make changes to your address below, please update your address in the account
+                  settings after you have completed thisfiling to ensure your information is up to date.
+                </p>
+              </v-card>
+
               <v-form
                 ref="addPersonOrgForm"
                 class="appoint-form"
+                :class="{ 'mt-8': isCompletingParty && !isRoleStaff }"
```

Comment:
I didn't want to add bottom margin to the conditional element above so I did this instead.